karaf-features-google/src/main/feature/feature.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="google-${project.version}"
+    xmlns="http://karaf.apache.org/xmlns/features/v1.5.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.5.0 http://karaf.apache.org/xmlns/features/v1.5.0">
+
+    
+    <feature name="jimfs" version="1.1" description="Google JimFS" install="false">
+        <bundle>mvn:com.ibm.icu/icu4j/49.1</bundle>
+        <feature>guava-18</feature>
+        <bundle>mvn:com.google.jimfs/jimfs/1.1</bundle>
+    </feature>
+
+    <feature name="guava" version="20.0" description="Google Guava 20" install="false">
+        <feature>guava-20</feature>
+    </feature>
+
+    <feature name="guava-18" version="18.0" description="Google Guava 18" install="false">
```

Comment:
Do we really need 10 different guava versions? :O

Reply:
yes :)